CollectionsInterface/Exercise3/src/Main.java

```diff
@@ -1,6 +1,24 @@
-public class Main {
+import java.util.ArrayList;
 
+public class Main {
+	
+	public static void removeEvenLenght(ArrayList<String> strings){
+        for(int i = 0; i<strings.size(); i++){
+            int length = strings.get(i).length();
+            if(length % 2 == 0)
+                strings.remove(strings.get(i));
+        }
+    }
+	
     public static void main(String[] args) {
-	// write your code here
+        ArrayList<String>strings = new ArrayList<>();
+        strings.add("Federico");
+        strings.add("Ernesto");
+        strings.add("Jazmin");
+        strings.add("Jorge");
+        strings.add("Javier");
```

Comment:
Proba con este input a ver que pasa... :thinking:
```suggestion
        strings.add("Federico");
        strings.add("Javier");
        strings.add("Ernesto");
        strings.add("Jazmin");
        strings.add("Jorge");
```

Reply:
Uh "Javier" no se me borra, mañana voy a tratar de arreglarlo entonces.

Reply:
Ahí arreglé todo, no lo había podido hacer porque estuve haciendo el 6 y después tuve unos problemas con git